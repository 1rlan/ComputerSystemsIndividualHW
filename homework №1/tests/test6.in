10 594722365
1710695898 594722365 1357921493 1834429687 1665882921 638635049 2112747170 1376444300 1046600257 166699390 
