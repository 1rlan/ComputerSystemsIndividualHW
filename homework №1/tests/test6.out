1710695898 1357921493 1834429687 1665882921 638635049 2112747170 1376444300 1046600257 166699390
<<<<<<< HEAD
=======

>>>>>>> master
